### PR TITLE
fix: hydra allow for specifying topologySpreadConstraints

### PR DESCRIPTION
**Pull request by DavidSpek**
## Related Issue or Design Document

This small PR allows for specifying the [topologySpreadConstraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) for the Hydra deployment. This is useful when trying to spread the Hydra pods across availability zones.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/hydra/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: "v2.0.1"
 description: A Helm chart for deploying ORY Hydra in Kubernetes
 name: hydra
 icon: https://raw.githubusercontent.com/ory/docs/master/docs/static/img/logo-hydra.svg
-version: 0.26.1
+version: 0.26.2
```
Comment (Demonsthere) [inline helm/charts/hydra/Chart.yaml:6]: Please do not edit that, as it is auto-generated :)

**Review by Demonsthere — COMMENTED** [review]
Hello there :) 
A nice, easy feature, see no reason why not to include it. Would you bo so kind to expand the feature to the rest of the charts?

File: helm/charts/hydra/Chart.yaml (hunk 1 above)
Comment (DavidSpek) [reply helm/charts/hydra/Chart.yaml]: Done

**Comment by DavidSpek** [comment]
@Demonsthere I've added the TopologySpreadConstraints to all the other charts now as well.

**Review by Demonsthere — APPROVED** [review]
Great job 👍

**Pull request merged**